arches/install/arches-templates/MANIFEST.in

```diff
@@ -1 +1 @@
-graft *
```

Comment:
The `[tool.setuptools]` setting in pyproject.toml already provides a sane default. `graft *` was picking up node modules.